mirar/processors/photometry/psf_photometry.py

```diff
@@ -143,15 +148,3 @@ def _apply_to_sources(
             source_table.set_data(candidate_table)
 
         return batch
-
-    def check_prerequisites(
```

Comment:
PSFex is not required at all, and having PSFE somewhere in the pipeline doesn't guarantee that psf phot will work (e.g. for a difference image). So I think this can go